kart/raster/metadata_util.py

```diff
@@ -1,21 +1,32 @@
+import logging
+from pathlib import Path
+from xml.dom import minidom
+from xml.dom.minidom import Node
+
 from kart.crs_util import normalise_wkt
 from kart.geometry import ring_as_wkt
 from kart.list_of_conflicts import ListOfConflicts
 from kart.schema import Schema, ColumnSchema
 
 
+L = logging.getLogger("kart.raster.metadata_util")
+
+
 def rewrite_and_merge_metadata(tile_metadata_list):
     """
     Given a list of tile metadata, merges the parts we expect to be homogenous into a single piece of tile metadata in
     the same format that describes the whole list.
     """
     # TODO - this will get more complicated as we add support for convert-to-COG.
     result = {}
+    all_keys = set().union(*tile_metadata_list)
```

Comment:
initially I thought this was just `set(tile_metadata_list)` but then I noticed the `*` and realised maybe `union` can take more than one argument - is this unioning the contents of sublists together?

if so, maybe make that more obvious:
```suggestion
    all_keys = set()
    for tm in tile_metadata_list:
        all_keys.update(tm)
```

Reply:
Done